README.rst

```diff
@@ -355,7 +381,7 @@ Many examples can be found under the `"Examples" <https://github.com/NVIDIA/NeMo
 Contributing
 ------------
 
-We welcome community contributions! Please refer to the  `CONTRIBUTING.md <https://github.com/NVIDIA/NeMo/blob/stable/CONTRIBUTING.md>`_ CONTRIBUTING.md for the process.
+We welcome community contributions! Please refer to `CONTRIBUTING.md <https://github.com/NVIDIA/NeMo/blob/stable/CONTRIBUTING.md>`_ for the process.
```

Comment:
Can you drop the publications.md, and add a link to the publications page instead ? Plus also link to the gh pages src GitHub readme to allow users to add their own results.

Reply:
Are you saying to delete the `PUBLICATIONS.md` file?

Reply:
(Confirmed we want to delete the `PUBLICATIONS.md` file)